test/functional/api/v2/hosts_controller_test.rb

```diff
@@ -100,6 +100,34 @@ def expect_attribute_modifier(modifier_class, args)
     assert_response :created
   end
 
+  test "should create host with default pxe_loader" do
```

Comment:
https://github.com/theforeman/foreman/tree/develop/test/functional/shared , sorry for not being clear

Like https://github.com/theforeman/foreman/search?utf8=%E2%9C%93&q=reporthostpermissionstest